src/org/team4909/bionic/utils/subsystems/BionicVelocitySubsystem.java

```diff
@@ -1,20 +1,46 @@
 package org.team4909.bionic.utils.subsystems;
 
 import org.team4909.bionic.utils.setpoints.VelocitySetpoint;
-
 import edu.wpi.first.wpilibj.SpeedController;
 import edu.wpi.first.wpilibj.MotorSafety;
-
 import edu.wpi.first.wpilibj.command.Subsystem;
+import com.ctre.CANTalon;
+import com.ctre.CANTalon.FeedbackDevice;
+import com.ctre.CANTalon.TalonControlMode;
 
 public abstract class BionicVelocitySubsystem extends Subsystem {
+	
 	public SpeedController speedController;
+	CANTalon _talon = new CANTalon(0);
 	
 	public BionicVelocitySubsystem(SpeedController speedController) {
 		this.speedController = speedController;
 		
 		((MotorSafety) this.speedController).setSafetyEnabled(true);
```

Comment:
you can get rid of the motor safety function, I was going to remove that from the voltage subsystem later also.